terraform/deploy/terraform.tfvars.j2

```diff
@@ -1,10 +1,16 @@
 assume_role = "administrator"
 parent_domain_name    = "{{dataworks_domain_name}}"
 whitelist_cidr_blocks = ["{{ucfs['team_cidr_block']}}"]
+ui_access_cidr_block = "{{ucfs['team_cidr_block']}}"
```

Comment:
how/why is this different to `whitelist_cidr_blocks`?

Reply:
The place that uses this can't take a list of strings. I contemplated changing `whitelist_cidr_blocks` but bottled it.

Reply:
This is fixed now.